### PR TITLE
Flatten model modules, remove the Support::Requests::Anonymous namespace

### PR DESCRIPTION
**Pull request by benilovj**
This PR doesn't make any functional modifications to the API, only the following refactorings:

- the `Support::Requests::Anonymous` module has been removed because
it made the directory structure hard to work with and caused lots of
extra nuisance when using the rails console
- require statements featuring models located in `app/models` have been
removed (Rails autoloads those classes automatically)
- two new model files have been created (`TopUrls`, `FeedbackCounts`)
for classes that were housed within the file belonging to another class.
It's best practice for each ruby file to only contain one class.
- the database has been migrated to remove module on anonymous contacts STI type.
This brings the single table inheritance `type` column on the
anonymous_contacts table in line with the flattened model classes.

/cc @binaryberry @mikejustdoit @boffbowsh

### CODE REVIEW

File: app/models/anonymous_contact.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
+require 'field_which_may_contain_personal_information'
```
Comment (boffbowsh) [inline app/models/anonymous_contact.rb]: Is `lib` not included in autoload?

**Comment by boffbowsh** [comment]
Looks good apart from tiny comment :+1:

File: app/models/anonymous_contact.rb (hunk 1 above)
Comment (benilovj) [inline app/models/anonymous_contact.rb]: It seems like if I remove this, then it starts failing (either missing constants when running the specs  or from rake tasks (eg `anonymous_feedback_deduplication:recent`) so I've left it in there for now.

Nonetheless, I fixed a couple of places where I still had `support/requests/anonymous/...` mentioned, which should mean that all rake tasks work now.

**Comment by benilovj** [comment]
This should be ready for another look.

**Comment by boffbowsh** [comment]
Looks good :+1:

**Pull request merged**